master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -1,132 +1,63 @@
 ---
-title: Configuring calicoctl - etcdv2 datastore 
-layout: docwithnav
+title: Connecting calicoctl to etcd
 ---
 
-This document covers the configuration options for calicoctl when using an etcdv2 datastore.
-
-## Configuration file 
+## Configuration File
 
 By default `calicoctl` looks for a configuration file at `/etc/calico/calicoctl.cfg`.
 
 The file location may be overridden using the `--config` option on commands that required
 datastore access.
 
-The config file is a yaml map, with the following supported fields: 
-
-| Field           | Description                                | Examples
-|-----------------|--------------------------------------------|----------
-| etcdEndpoints   | A comma separated list of etcd endpoints.  | http://etcd1:2379 
-| etcdUsername    | Username for RBAC (optional)               | "user" 
-| etcdPassword    | Password for the given username (optional) | "password"
-| etcdKeyFile     | Path to the etcd key file (optional)       | /etc/calico/key.pem
-| etcdCertFile    | Path to the etcd client cert (optional)    | /etc/calico/cert.pem
-| etcdCACertFile  | Path to the etcd CA file (optional)        | /etc/calico/ca.pem
-
-> **NOTE** 
-> 
-> If you are running with TLS enabled, ensure your endpoint addresses use https
-
-Each of the above configuration options can also be set through environment variables.  For example,
-`etcdEndpoints` can also be represented as `ETCD_ENDPOINTS`.
-
-#### Example configuration file
+The following is an example `calicoctl.cfg`, which lists all supported fields:
```

Comment:
I don't really like this approach to documenting our configuration options.  Examples are not reference documentation.  They can't express all the stuff we might want to express - we should be using tables which outline the valid configuration options rather than examples.